newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -323,17 +326,43 @@ export default class AuthenticatedUserProvider extends React.Component<
   };
 
   _fetchUserProfile = async (
-    options: ?{ dontNotifyAboutEmailVerification?: boolean }
+    options: ?{
+      dontNotifyAboutEmailVerification?: boolean,
+      resetState?: boolean,
```

Comment:
maybe add a comm next to this prop, to understand when this is helpful.
Like "helpful when a user goes from unauthenticated to authenticated, so we can track when the profile attributes are finished loading (projects, assets...)"